tv_apps/smart-system/test/marionette/app_install_dialog_test.js

```diff
@@ -50,10 +47,9 @@ marionette.skip('app install manager tests', function() {
 
   setup(function() {
     system = client.loader.getAppClass('smart-system', 'system', 'tv_apps');
+    system.waitForStartup();
+    system.waitForFullyLoaded();
     appInstaller = new AppInstaller(client);
```

Comment:
Doing works after system.waitForStartup and system.waitForFullyLoaded can help to avoid unexpected 'before each' failure